docs/meshstack.billing-configuration.md

```diff
@@ -377,13 +383,15 @@ let example
         , "paymentAmount"
         ]
       , period-offset-days = 10
+      , first-period = "2021-01-01T00:00:00Z"
```

Comment:
d: You removed the config for VW and EnBW now completely? But at VW it must be set to "2020-12-01" and at enbw it must remain at "2020-01-01", because otherwise we will generate more periods of the past, which we don't want. Why did you remove the configs completely? Was there some confusion coming up due to our recent discussions around it?

Reply:
Sorry for the confusion; this is the example part in the docs only. In the original PR https://github.com/meshcloud/meshfed-release/pull/840 VW is changed to 2020-12-01 and EnBW will remain using the default `application.yml` which is still on 2020-01-01.

Reply:
I updated the example to also be similar to `application.yml` values